assets/src/scripts/monitoring-location/components/hydrograph/legend.test.js

```diff
@@ -36,56 +34,19 @@ describe('monitoring-location/components/hydrograph/legend module', () => {
                         approved: false,
                         estimated: false
                     }]
-                },
```

Comment:
This test state had a bunch of data that didn't do anything, and was confusing me, so it trimmed it a bit.

Reply:
Anything around the Redux state in the tests is going to be changing dramatically anyways. Trimming unneeded stuff is always welcome!